lib/helpers/helpers-html.js

```diff
@@ -157,6 +157,16 @@ module.exports.register = function (Handlebars, opts) {
       }
       figure += '</figure>';
       return new Handlebars.SafeString(figure);
+    },
+
+    /**
+     * {{#striptags}}{{{data}}}{{{//striptags}}}
+     * Remove all HTML tags from the nested context
+     * @param  {[type]} options [description]
+     * @return {[type]}         Text with HTML tags removed
+     */
+    striptags: function (options) {
```

Comment:
@doowb @jonschlinkert is options right her or should it be context? I never know